flake.nix

```diff
@@ -196,26 +216,31 @@
 
       # Ply x Plutarch
       ply-plutarch = rec {
-        ghcVersion = "921";
+        ghcVersion = "923";
         compiler-nix-name = "ghc${ghcVersion}";
 
         projectFor = system:
           let
-            pkgs = nixpkgsFor system;
-            pkgs' = nixpkgsFor' system;
-            stdDevEnv = mkDevEnv system;
-            hls = pkgs.haskell-language-server.override { supportedGhcVersions = [ ghcVersion ]; };
+            pkgs = import plutarch.inputs.nixpkgs {
+              inherit system;
+              inherit (plutarch.inputs.haskell-nix) config;
+              overlays = [
+                plutarch.inputs.haskell-nix.overlay
+                (import "${plutarch.inputs.iohk-nix}/overlays/crypto")
+              ];
+            };
+            stdDevEnv = mkDevEnv system; # TODO: parametrize with pkgs rather?
+            hls = (plutarch.hlsFor compiler-nix-name system);
+            myPlutarchHackages = plutarch.inputs.haskell-nix-extra-hackage.mkHackagesFor system compiler-nix-name [
+              "${inputs.plutarch}"
+            ];
```

Comment:
@TotallyNotChase see here I break the nix pattern by just using plutarch.inputs and not orchestrating that at the Flake inputs. You ok with that?

Reply:
As long as it works, it can stay